src/app/pages/SearchResultsPage/ResultListFrame.tsx

```diff
@@ -0,0 +1,35 @@
+import Box from '@mui/material/Box'
+import { styled } from '@mui/material/styles'
+import useMediaQuery from '@mui/material/useMediaQuery'
+
+export const ResultListFrame = styled(Box)(({ theme }) => {
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'))
+  return {
+    marginBottom: 20,
+    border: isMobile ? 'none' : `solid 15px ${theme.palette.layout.networkBubbleBorder}`,
+    background: theme.palette.layout.networkBubbleBackground,
+    borderRadius: 12,
+    boxShadow: '-20px 4px 40px rgba(34, 47, 63, 0.24)',
+    '&& .MuiCard-root': {
+      borderTopLeftRadius: 0,
+      borderTopRightRadius: 0,
+      paddingLeft: 64,
+      paddingRight: 64,
+      background: theme.palette.layout.networkBubbleBackground,
+    },
+    '&& .MuiCard-root > .MuiBox-root': {
```

Comment:
Could use `cardClasses.*`

Reply:
Done. Too bad we don't have a similar thing for `MuiBox` ... or do we?